src/components/dialog/dialog.js

```diff
@@ -221,13 +221,14 @@ class Dialog extends Modal {
 
   get closeIcon() {
     if (this.props.showCloseIcon) {
-      return <Icon
-        className="carbon-dialog__close"
+      return (<Icon
```

Comment:
Yes I think the parens were added automatically when I ran eslint with `--fix` but it obviously didn't change the indentation.